assets/src/components/detours/diversionPanel.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactNode } from "react"
+import { RoutePill } from "../routePill"
+
+export interface DiversionPanelProps {
+  directions?: ReactNode
+  missedStops?: ReactNode
+  routeName: string
+  routeDescription: string
+  routeOrigin: string
+  routeDirection: string
+}
+
+export const DiversionPanel = ({
+  directions,
+  missedStops,
+  routeName,
+  routeDescription,
+  routeOrigin,
+  routeDirection,
+}: DiversionPanelProps) => (
+  <article className="c-diversion-panel h-100 bg-light border-end">
+    <header className="c-diversion-panel__header align-items-center border-bottom d-flex justify-content-around px-3">
```

Comment:
General question: How do we decide what of these styles should live in `_*.scss` and what should go directly here?

Reply:
Good question!
I don't think we have an established practice yet. Part of me wants to reduce the amount of classes and use scss `@extend` but I think it may be better to do it within the DOM as it is?

So far I was adding classes "ad-hoc" as needed to get close to the styles in the Figma, and anything I couldn't do with bootstrap classes (bespoke grid) I did in the `_*.scss` file.

If you have any input on how to approach this I'd be happy to, at the very least, consider it!

Reply:
How come it would be better to do it within the DOM? 

Right now we sort of have a hybrid between SCSS and utility classes, which is a bit confusing to me. I dunno that I have a strong opinion about which direction we should go, but I do think that we'd be better off picking one approach and going all in on that.

I'll spend some time thinking about which approach I personally would prefer.

Aaaaand since this seems like a "problem" (at least a feature) throughout our codebase, I don't think we need to solve it now - another refactor opportunity!

Reply:
I mean... moving to Tailwind is an option :)

(I personally DO like Tailwind and its utility class concept - it makes things very flexible, but also React components [or Phoenix components if we ever go that route] give us plenty of reusability)

Curious to see that diff!

Reply:
I did immediately run into an issue that wouldn't happen if all the classes were in the HTML.
Right now we don't have typography imported, which means styles like `.h3` are not available, and SCSS fails to build when doing `@extend .h3` because the style is missing.

Reply:
it does seem like I can use `!optional`` to get around this

Reply:
I'm not sure if this is what you had imagined, but here it is rewritten using scss @extend and bootstrap. I opted to keep using bootstrap and `@extend` so that any design tokens edited in the bootstrap config would automatically apply here.
dda154f6d5c78d8aded644836837b93fc7474267